package.json

```diff
@@ -41,7 +41,6 @@
     "ember-cli-babel": "^7.26.3",
     "ember-cli-htmlbars": "^5.7.1",
     "ember-get-config": "^0.2.4 || ^0.3.0",
-    "ember-on-modifier": "^1.0.0",
```

Comment:
so this would be a breaking change, it would require the end user to change their dependencies if they don't have it already installed. Since it's `Completely inert when running ember-source 3.11 or higher` I think it's fine to include in the dependencies 👍

Reply:
The fact that it prints a message shows it is not actually inert 😄 but I get where you're going with this.

Reply:
Updated 🙂